source/openstack/guests.rst

```diff
@@ -1,23 +1,34 @@
-.. _aarch64-openstack-guests:
+.. _openstack-guests:
 
-Building Openstack AARCH64 (arm64) Guests
-=========================================
+Building OpenStack Guests
+=========================
 
-This documentation covers the steps needed to build an Openstack guests on AARCH64 (arm64) using a tool called
-`packer`. In addition, we include documentation which covers how to build the images manually in case you run into
-issues with packer.
+This documentation covers the steps needed to build OpenStack guests using a tool called `packer`. In addition,
+we include documentation which covers how to build the images manually in case you run into issues with packer.
 
 Current Images
 --------------
 
-We are currently maintaining AARCH64 (arm) images.
+We are currently maintaining these images:
 
-- CentOS 7
-- Debian 10
-- Fedora 31
-- Ubuntu 18.04
+.. csv-table::
+   :file: ./csv/images.csv
+   :widths: 40, 15, 15, 15, 15
+   :header-rows: 1
 
-If there are any images you would like us to create, please let us know.
+.. note:: We are currently fixing some issues with our Ubuntu 20.04 images.
```

Comment:
I'd rather we convert this into a [footnote](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#footnotes). Ideally you'd put the following in the csv file:
```
"Ubuntu 20.04 [#ubuntu]_","","","",""
```
And then this gets moved to the bottom of this file and turns into:
```rst
.. rubric:: Footnotes

.. [#ubuntu] We are currently fixing some issues with our Ubuntu 20.04 images.
```